src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -31,6 +31,30 @@
             "\"Since Date\" cannot be later than \"Until Date\"";
     private static final Path EMPTY_PATH = Paths.get("");
 
+    private static final String FULL_PREFIX = "--";
+    private static final String ALIAS_PREFIX = "-";
+
+    private static final String HELP_FLAG = "help";
+    private static final String HELP_FLAG_ALIAS = "h";
+    private static final String CONFIG_FLAG = "config";
+    private static final String CONFIG_FLAG_ALIAS = "c";
+    private static final String REPO_FLAG = "repos";
+    private static final String REPO_FLAG_ALT = "repo";
+    private static final String REPO_FLAG_ALIAS = "r";
+    private static final String VIEW_FLAG = "view";
+    private static final String VIEW_FLAG_ALIAS = "v";
+    private static final String OUTPUT_FLAG = "output";
+    private static final String OUTPUT_FLAG_ALIAS = "o";
+    private static final String SINCE_FLAG = "since";
+    private static final String SINCE_FLAG_ALIAS = "s";
+    private static final String UNTIL_FLAG = "until";
+    private static final String UNTIL_FLAG_ALIAS = "u";
+    private static final String FORMAT_FLAG = "formats";
+    private static final String FORMAT_FLAG_ALIAS = "f";
+    private static final String IGNORE_FLAG = "ignore-standalone-config";
+    private static final String IGNORE_FLAG_ALIAS = "isac";
```

Comment:
As we may have much more CLI arguments in the future,  we may have some args with same single letter alias, maybe we just use -isac instead of -i because it is already an abstraction?

Reply:
Depends on how closely and faithfully are we sticking to POSIX standards. 🤔 

In POSIX, the usage of single letter can be together `i.e.` `-abcd` is equivalent as `-a -b -c -d`.

You won't run out of letters to use, but this may be the reason why single letter options are non-intuitive. 🤣